ImageBuilderRAM/README.md

```diff
@@ -0,0 +1,32 @@
+# EC2 Image Builder - Image Sharing via RAM for Dynamic Versioning
+
+EC2 Image Builder Allows customers to build an image pipeline which automates their image creation process starting from a source image installing custom software packages and security patches, the created image can then be shared across regions and other accounts.
+The pipeline can be automated to run on a schedule if there is an available update of the source image or any of the build components 
+
+
+![EC2 Image Builder Flow](./EC2ImageBuilder.png)
+
+Customers who have a dedicated team creating Images via EC2ImageBuilder service usually share the images via ImageBuilder service from a shared account.
```

Comment:
Please change `usually` to `commonly` or similar. We shouldn't imply what we believe is being done across most of our customers.